cluster-autoscaler/cloudprovider/oci/README.md

```diff
@@ -122,6 +140,10 @@ Configuration via environment-variables:
 - `OCI_REFRESH_INTERVAL` - Optional refresh interval to sync internal cache with OCI API defaults to `2m`.
 - `OCI_USE_NON_POOL_MEMBER_ANNOTATION` - Optional. If true, the node will be annotated as non-pool-member if it doesn't belong to any instance pool and the time-consuming instance pool lookup will be skipped.
 
+#### Node Pool environment-variables
+
+XXX
```

Comment:
Is `XXX` intended to mean that the node pool does not support any environment variables? Or was this a TODO that didn't get updated?

As far as I can tell, _both_ implementations support `OCI_USE_INSTANCE_PRINCIPAL` and `OCI_REFRESH_INTERVAL` environment variables. I added another comment asking whether if the call to `InstancePrincipalConfigurationProviderForRegion` in `oci_instance_pool_manager.go` could be replaced with `InstancePrincipalConfigurationProvider`. If so, the two implementations would behave similarly and the need for the `OCI_REGION` environment variable could go away. That would potentially leave only `OCI_USE_NON_POOL_MEMBER_ANNOTATION` as the only environment variable that only had use in one implementation. The others would be common to both.

Reply:
We've since updated this to have

```
#### Environment-variables applicable _only_ for Node Pools
n/a
```